sound/soc/sof/core.c

```diff
@@ -304,7 +325,10 @@ static int sof_probe(struct platform_device *pdev)
 			ret);
 		goto fw_run_err;
 	}
-
+#else
+	sdev->vfe = plat_data->vdev->priv;
+	sdev->vfe->sdev = sdev;
+#endif
```

Comment:
So the above ifdef could be replaced with if (sdev->is_vbe) and then the assignement above could be static inline within the header VBE section.  Just trying to eliminate all the ifdefs in common code and keeping it to header and one other place.